src/lib.rs

```diff
@@ -66,23 +66,26 @@
 
 #![allow(clippy::result_large_err)]
 
-use std::collections::HashMap;
 use std::fmt;
 use std::num::TryFromIntError;
 use std::time::Duration;
+use std::{collections::HashMap, marker::PhantomData};
 
 pub mod api;
 
 #[cfg(feature = "async")]
 pub mod r#async;
 #[cfg(feature = "blocking")]
 pub mod blocking;
+#[cfg(feature = "async")]
+pub mod sleeper;
```

Comment:
It's failing to build when using `--no-default-features`, in order to use only `blocking` client.

```suggestion
pub mod sleeper;
```